web/handlers.go

```diff
@@ -86,6 +86,53 @@ type Handler struct {
 	cspShaDirective string
 }
 
+func GenerateDevCSP(c Context) string {
+	// Add unsafe-eval to the content security policy for faster source maps in development mode
+	devCSPMap := make(map[string]bool)
+	if model.BuildNumber == "dev" {
+		devCSPMap["unsafe-eval"] = true
+	}
+
+	// Add unsafe-inline to unlock extensions like React & Redux DevTools in Firefox
+	// see https://github.com/reduxjs/redux-devtools/issues/380
+	if model.BuildNumber == "dev" {
+		devCSPMap["unsafe-inline"] = true
+	}
+
+	// Add supported flags for debugging during development, even if not on a dev build.
+	for _, devFlagKVStr := range strings.Split(*c.App.Config().ServiceSettings.DeveloperFlags, ",") {
+		devFlagKVSplit := strings.SplitN(devFlagKVStr, "=", 2)
+		if len(devFlagKVSplit) != 2 {
+			c.Logger.Warn("Unable to parse developer flag", mlog.String("developer_flag", devFlagKVStr))
+			continue
+		}
+		devFlagKey := devFlagKVSplit[0]
+		devFlagValue := devFlagKVSplit[1]
+
+		// Ignore disabled keys
+		if devFlagValue != "true" {
+			continue
+		}
+
+		// Honour only supported keys
+		switch devFlagKey {
+		case "unsafe-eval", "unsafe-inline":
+			devCSPMap[devFlagKey] = true
+		default:
+			c.Logger.Warn("Unrecognized developer flag", mlog.String("developer_flag", devFlagKVStr))
+		}
+	}
+	var devCSP string
+	supportedCSPFlags := []string{"unsafe-eval", "unsafe-inline"}
+	for _, devCSPFlag := range supportedCSPFlags {
+		if _, ok := devCSPMap[devCSPFlag]; ok {
```

Comment:
There is no need for `_, ok`. We can directly write `if devCSPMap[devCSPFlag] {`. If the key doesn't exist, it will return false.